worker/options.go

```diff
@@ -28,7 +29,7 @@ func calculateMaxExecsJitter(maxExecs, jitter uint64) uint64 {
 		return 0
 	}
 
-	percent := rand.Intn(int(jitter))
+	percent := rand.Intn(int(jitter)) //nolint:gosec
```

Comment:
Do not hide warnings 😃 
You should use `crypto/rand` instead. IIRC, you may generate the random int like this:
```go
    rand.Int(rand.Reader, big.NewInt(27)).Int64();
```
Where `big` is `math/big`.

Reply:
btw I don't remember, might be `rand.Int` also returns an error, so, you would need and extra step to check it.

Reply:
I already wrote an implementation in crypto/rand, but I decided to try to suppress the check and pass the review)

I looked at the entire repository and found such suppressions. We definitely don't need a super random number at this point, but okay.